src/Persistence/Models/Control.cs

```diff
@@ -94,6 +94,8 @@ public int ZIndex
     [OnDeserialized]
     internal virtual void AfterDeserialize()
     {
+        // Is this dead code now? Looks like it's not being hit anymore after the schema changes?
```

Comment:
Will clean up this comment, but wanted to check whether this was dead code or not?

Reply:
@anpetroc can you chime in? I was trying to debug this method this morning, but it doesn't get called (at least not when running the tests). If that's the case, I'd say we can get rid of it

Reply:
It probably is dead code now as we are switching to some new object model hooks